### PR TITLE
Docs/migrate to docusarus

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

### Migrated to Docusaurus:

* Delete mdbooks
* Copied docs folder to docOps
* Initiated docusaurus
* Closed JSX tags that were causing compilation errors
* Moved images to docsOps/static
* Fixed broken links
* Improved slugs
* Removed HTML
* Created local copy images

**NO CONTENT/TEXT WAS EDITED, ONLY HOW WE SERVE IT**

## Links to any relevant issues

[Edit all text with our style guide](https://github.com/iotaledger/docs.iota.org/issues/2)

## Type of change
- Enhancement (a non-breaking change which adds functionality)
- Documentation Fix

## How the change has been tested

You can follow the docs/README.md file for running docusaurus.
The documentation was served and reviewed locally.  
The documentation is currently being served using GH pages at [https://lucas-tortora.github.io/goshimmer/](https://lucas-tortora.github.io/goshimmer/) . 

## Change checklist
- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: .github/deploy_docs_to_gh-pages.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+name: deploy
+
+on:
+  pull_request:
+    branches: [migrateToDocusaurus, dev, main]
```
Comment (huhn511) [inline .github/deploy_docs_to_gh-pages.yml:5]: change `dev` to ´develop´

File: .github/deploy_docs_to_gh-pages.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          npm ci
+          else
+          npm i
+          fi
+          npm run build
+      - name: Release to GitHub Pages
+        uses: iotaledger/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./docOps/build
```
Comment (huhn511) [inline .github/deploy_docs_to_gh-pages.yml:52]: add `cname: goshimmer.docs.iota.org`

File: docOps/docusaurus.config.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').DocusaurusConfig} */
+module.exports = {
+  title: 'Goshimmer',
```
Comment (huhn511) [inline docOps/docusaurus.config.js:6]: Change to `goShimmer`

**Review by huhn511 — CHANGES_REQUESTED** [review]
- [ ] remove doubled gh-action workflow
- [ ] change landing page content

**Comment by anistark** [comment]
I can see these variations of the name:
1. GoShimmer
2. goShimmer
3. Goshimmer
4. goshimmer

I think we should stick to 1 particular naming convention everywhere. I believe the official one is "GoShimmer". Or if we don't have an official one, let's settle for one.

**Review by capossele — APPROVED** [review]
Super awesome!

**Review by huhn511 — APPROVED** [review]
LGTM! 👍

**Pull request merged**